tests/output/libsh/libsh.h

```diff
@@ -0,0 +1 @@
+extern int libshared(void);
```

Comment:
GitHub's showing me a funny symbol saying there's no line at the end of the file here?

(not a *trailing* blank line - a `\n`character after `libshared(void);`)